src/presentation/http/router/noteSettings.test.ts

```diff
@@ -281,4 +281,61 @@ describe('NoteSettings API', () => {
 
     test.todo('Return 403 when user authorized, but not member of the team');
   });
+
+  describe('PATCH /note-teams/new-role/:newRole', () => {
+    test('Update team member role by user id and note id, with status code 200', async () => {
+      const response = await global.api?.fakeRequest({
```

Comment:
can you please add some comments clarifying what's happening in the test?

Reply:
done